install-guide.py

```diff
@@ -0,0 +1,342 @@
+    --hostname faceswap-cpu --name faceswap-cpu \\
+    -v {path}:/srv \\
+    -v /tmp/.X11-unix:/tmp/.X11-unix \\
+    -e DISPLAY=unix$DISPLAY \\
+    -e AUDIO_GID=`getent group audio | cut -d: -f3` \\
+    -e VIDEO_GID=`getent group video | cut -d: -f3` \\
+    -e GID=`id -g` \\
+    -e UID=`id -u` \\
+    faceswap-cpu
+
+
+4. Open a new terminal to run faceswap.py in /srv
+docker exec -it deepfakes-cpu bash
+'''.format(path=sys.path[0]))
+    INFO("That's all you need to do with a docker. Have fun.")
+
+def Tips_1_2():
+    INFO('''1. Install Docker
+https://www.docker.com/community-edition
+
+2. Install latest CUDA
+CUDA: https://developer.nvidia.com/cuda-downloads
+
+3. Install Nvidia-Docker
+https://github.com/NVIDIA/nvidia-docker
+
+4. Build Docker Image For Faceswap
+docker build -t deepfakes-gpu -f Dockerfile.gpu
+
+5. Mount faceswap volume and Run it
+# without gui. tools.py gui working.
+docker run -p 8888:8888 --hostname faceswap-gpu --name faceswap-gpu -v {path}:/srv faceswap-gpu
+
+# with gui. tools.py gui not working.
+docker run -p 8888:8888 \\
+    --hostname faceswap-gpu --name faceswap-gpu \\
+    -v {path}:/srv \\
+    -v /tmp/.X11-unix:/tmp/.X11-unix \\
+    -e DISPLAY=unix$DISPLAY \\
+    -e AUDIO_GID=`getent group audio | cut -d: -f3` \\
+    -e VIDEO_GID=`getent group video | cut -d: -f3` \\
+    -e GID=`id -g` \\
+    -e UID=`id -u` \\
+    faceswap-gpu
+
+6. Open a new terminal to interact with the project
+docker exec faceswap-gpu python /srv/tools.py gui
+'''.format(path=sys.path[0]))
+
+def Tips_2_1():
+    INFO('''Tensorflow has no official prebuilts for CUDA 9.1 currently.
+
+1. Install CUDA 9.0 and cuDNN
+CUDA: https://developer.nvidia.com/cuda-downloads
+cuDNN: https://developer.nvidia.com/rdp/cudnn-download (Add DLL to %PATH% in Windows)
+
+2. Install System Dependencies.
+In Windows:
+Install CMake x64: https://cmake.org/download/
+
+In Debian/Ubuntu, it's like:
+apt-get install -y cmake libsm6 libxrender1 libxext-dev python3-tk
+
+3. Install PIP requirements
+You may want to execute `chcp 866` in cmd line
+to fix Unicode issues on Windows when installing dependencies
+''')
+
+
+def Tips_2_2():
+    INFO('''1. Install System Dependencies.
+In Windows:
+Install CMake x64: https://cmake.org/download/
+
+In Debian/Ubuntu, it's like:
+apt-get install -y cmake libsm6 libxrender1 libxext-dev python3-tk
+
+2. Install PIP requirements
+You may want to execute `chcp 866` in cmd line 
+to fix Unicode issues on Windows when installing dependencies
+''')
+
+
+def Main():
+    global ENABLE_DOCKER, ENABLE_CUDA, CUDA_Version, OS_Version
+    Check_System()
+    Check_Python()
+    Check_PIP()
+    # ask questions
+    Enable_CUDA()
+    Enable_Docker()
+    # warn if nvidia-docker on non-linux system
+    if OS_Version[0] != "Linux" and ENABLE_DOCKER and ENABLE_CUDA:
+        WARNING("Nvidia-Docker is only supported on Linux.\r\nOnly CPU is supported in Docker for your system")
+        Use_Docker()

```

Comment:
Is this a builtin? Use_Docker doesn't otherwise exist as a function

Reply:
It is an embarrassing mistake. I renamed function Use_Docker() to Enable_Docker() but missed this one.